torch/csrc/jit/tensorexpr/kernel.cpp

```diff
@@ -505,6 +505,24 @@ void promoteInputs(std::vector<ExprHandle>& inputs, const int typeConstraints) {
   }
 }
 
+ExprHandle promoteIntegerToDefaultType(const ExprHandle& e) {
```

Comment:
This code was just moved from below to define it in appropriate namespace since this is now exposed through the header and used in other files.